### DISCUSSION TITLE
FAQ/

### DISCUSSION THREAD

**Discussion by giscus** [Air-duino/document#8] [comment]
# FAQ/

https://arduino.luatos.com/FAQ/

**Comment by liguichuan** [comment]
Air001 同时处理多串口通信，比如UART1用于输出调试，UART2用于接其他电子模块的串口获取数据，具体在Arduino中怎么处理？Serial.xx和Serial2.xx不行。就是第二个串口要如何使用。

**Reply by lishuzhou** [reply]
同问啊，折腾了几天不知道咋搞！！！

**Comment by lishuzhou** [comment]
Air001有两个USART，Arduino使用Serial跟Serial1没法区分，怎么使用多串口？

**Reply by chenxuuu** [reply]
https://arduino.luatos.com/library/uart.html#airmcu-%E4%B8%93%E7%94%A8api

**Reply by lishuzhou** [reply]
Air001 看芯片手册是有片上 rtc 资源的，Arduino 如何开发呢，有没有相关 demo？

**Comment by lishuzhou** [comment]
非常感谢❤️



---Original---
From: ***@***.***&gt;
Date: Fri, Jul 5, 2024 11:20 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [Air-duino/document] FAQ/ (Discussion #8)




 
https://arduino.luatos.com/library/uart.html#airmcu-%E4%B8%93%E7%94%A8api

**Comment by lishuzhou** [comment]
Air001 看芯片手册是有片上 rtc 资源的，Arduino 如何开发呢，有没有相关 demo？
